tests/conftest.py

```diff
@@ -202,6 +202,19 @@ def dnadiff_reports_dir() -> Path:
     return FIXTUREPATH / "dnadiff" / "targets" / "dnadiff_reports"
 
 
+@pytest.fixture
+def dir_dnadiff_matrices() -> Path:
+    """Directory containing dnadiff matrices."""
+    return FIXTUREPATH / "dnadiff" / "matrices"
+
+
+@pytest.fixture
+def dnadiff_matrices(dir_dnadiff_matrices: Path) -> list[str]:
+    """Target dnadiff matrices."""
+    reference_paths = (FIXTUREPATH / "dnadiff" / "matrices").glob("*.tsv")
+    return [dir_dnadiff_matrices / _.name for _ in reference_paths]
```

Comment:
Do you actually use the second feature ``dnadiff_matrices``?

Reply:
Good spot! I was planning to use it, but I switched to an alternative! I will remove this used fixture.